src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -274,29 +273,40 @@ class AbstractIntegrationDetailedView<
 
   //Returns the information about the integration description and features
   renderInformationCard() {
-    const {FeatureList} = getIntegrationFeatureGate();
+    const {IntegrationDirectoryFeatureList} = getIntegrationFeatureGate();
 
     return (
       <React.Fragment>
-        <Description dangerouslySetInnerHTML={{__html: marked(this.description)}} />
-        <FeatureList {...this.featureProps} provider={{key: this.integrationSlug}} />
-        {this.renderPermissions()}
-        <Metadata>
-          {!!this.author && <AuthorName>{t('By %s', this.author)}</AuthorName>}
-          <div>
+        <Flex>
+          <FlexContainer>
+            <Description dangerouslySetInnerHTML={{__html: marked(this.description)}} />
+            <IntegrationDirectoryFeatureList
+              {...this.featureProps}
+              provider={{key: this.props.params.integrationSlug}}
+            />
+            {this.renderPermissions()}
+            {this.alerts.map((alert, i) => (
+              <Alert key={i} type={alert.type} icon={alert.icon}>
+                <span
+                  dangerouslySetInnerHTML={{__html: singleLineRenderer(alert.text)}}
+                />
+              </Alert>
+            ))}
+          </FlexContainer>
+          <Metadata>
+            {!!this.author && (
+              <div>
+                <CreatedContainer>{t('Created By')}</CreatedContainer>
+                <AuthorName>{t(this.author)}</AuthorName>
```

Comment:
@NisanthanNanthakumar this is where you need to use the `tct` translation and use the author as a variable